accepted/2024/[TS]PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,213 @@
+                        "licenseExpression": "MIT",
+                        "licenseUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3/license",
+                        "readmeUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3#show-readme-container",
+                        //New field
+                        "readmeFileUrl": "https://apidev.nugettest.org/v3/flatcontainer/newtonsoft.json/13.0.3/readme",
+                        "listed": true,
+                        "minClientVersion": "2.12",
+                        "packageContent": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/newtonsoft.json.13.0.3.nupkg",
+                        "projectUrl": "https://www.newtonsoft.com/json",
+                        "published": "2023-04-25T14:48:53.817+00:00",
+                        "requireLicenseAcceptance": false,
+                        "summary": "",
+                        "tags": [
+                            "json"
+                        ],
+                        "title": "Json.NET",
+                        "version": "13.0.3"
+                    },
+                    "packageContent": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/newtonsoft.json.13.0.3.nupkg",
+                    "registration": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/index.json"
+                },
+                ...
+            ],
+            ...
+        }
+    ],
+    ...
+}
+```
+##### ReadmeUriTemplate Resource Type
+A new resource `ReadmeUriTemplate` similar to the [ReportAbuseUriTemplate](https://learn.microsoft.com/en-us/nuget/api/report-abuse-resource) resource type which will include a url definition for downloading the README.
+
+```json
+{
+    "version": "3.0.0",
+    "resources": [
+        ...,
+        {
+            "@id": "https://apidev.nugettest.org/v3/flatcontainer/{lower_id}/{lower_version}/readme",
+            "@type": "ReadmeUriTemplate/6.13.0"
+        },
+        ...
+    ]
+}
+```
+In code we would take the id provided and replace the {lower_id} and {lower_version} in the strings with the package we are trying to get the README for.
+
+### Technical explanation
+#### Rendering Markdown
+
+We will use the [Microsoft.VisualStudio.Markdown.Platform](https://dev.azure.com/azure-public/vside/_artifacts/feed/vs-impl/NuGet/Microsoft.VisualStudio.Markdown.Platform/overview/18.0.39-preview-g659b28ccd8) package to render the README in the IDE. This will allow us to leverage a centralized tool for rendering markdown in the IDE. 
+
+A new instance of the [IMarkdownPreview](https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=%2Fsrc%2FProductivity%2FMarkdownLanguageService%2FImpl%2FMarkdown.Platform%2FPreview%2FIMarkdownPreview.cs) can be created using [PreviewBuilder](https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=/src/Productivity/MarkdownLanguageService/Impl/Markdown.Platform/Preview/PreviewBuilder.cs).
```

Comment:
This feels like implementation detail that doesn't need to be a specification document. Plus our specs are publicly available, so links to closed source doesn't feel appropriate to me.

Reply:
I removed the links and just skimmed it down to say we're using the control